skxray/core.py

```diff
@@ -631,45 +627,49 @@ def pixel_to_radius(shape, calibrated_center, pixel_size=None):
         pixel_size = (1, 1)
 
     X, Y = np.meshgrid(pixel_size[1] * (np.arange(shape[1]) -
-                                        calibrated_center[1]),
+                                        center[1]),
                        pixel_size[0] * (np.arange(shape[0]) -
-                                        calibrated_center[0]))
+                                        center[0]))
     return np.sqrt(X*X + Y*Y)
 
 
-def pixel_to_phi(shape, calibrated_center, pixel_size=None):
+def angle_grid(center, shape, pixel_size=None):
     """
-    Converts pixel positions to :math:`\\phi`, the angle from vertical.
+    Make a grid of angular positions.
+
+    Read note for our conventions here -- there be dragons!
 
     Parameters
     ----------
-    shape : tuple
-        The shape of the image (nrow, ncol) to warp
-        the coordinates of
+    center : tuple
+        point in image where r=0; may be a float giving subpixel precision.
+        Order is (rr, cc).
 
-    calibrated_center : tuple
-        The center in pixels-units (row, col)
-
-    pixel_size : tuple, optional
-        The size of a pixel (really the pitch) in real units. (height, width).
-
-        Defaults to 1 pixel/pixel if not specified.
+    shape: tuple
+        Image shape which is used to determine the maximum extent of output
+        pixel coordinates. Order is (rr, cc).
 
     Returns
     -------
-    phi : array
-        :math:`\\phi`, the angle from the vertical axis.
-        :math:`\\phi \\el [-\pi, \pi]`
+    agrid : array
+        angular position (in radians) of each array element in range [-pi, pi]
+
+    Note
+    ----
+    :math:`\\theta`, the counter-clockwise angle from the positive x axis
+    :math:`\\theta \\el [-\pi, \pi]`. Postivie y is downward, so 45-degrees
```

Comment:
spelling + 'In array indexing and the conventional axes for images (origin in upper left), positive ``y`` is downward)